examples/quickstart/src/main/java/quickstart/Quickstart.java

```diff
@@ -105,6 +111,20 @@ public static void main(String[] args) {
             e.getCauses().forEach( cause -> println("\t" + cause.getSummary()));
             throw e;
         }
+        finally {
+            // cleanup
+
+            // deactivate, delete user
+            if (user != null && user.getStatus() != UserStatus.DEPROVISIONED) {
+                user.deactivate();
+            }
+            user.delete();
```

Comment:
If user is null here this delete will fail

Reply:
addressed this.